src/main.c

```diff
@@ -16,8 +16,17 @@
 #include <stdio.h>      /* printf */
 #include <locale.h>     /* setlocale */
 #include <string.h>     /* strcmp */
+#include <termios.h>
```

Comment:
If we use ioctl, shouldn't we test if we build the program for a unix-like system? So we would keep the windows support. 
e.g.:
```c
#ifdef __unix__
// ..
#endif // __unix__
```